app/AcquireIntersonArrayRF.cxx

```diff
@@ -121,18 +121,18 @@ int main( int argc, char * argv[] )
   ContainerType container;
 
   const int height = hwControls.GetLinesPerArray();
+  std::cout << "Lines per array: " << height << std::endl;
   const int width = ContainerType::MAX_RFSAMPLES;
+  std::cout << "Max RF samples: " << width << std::endl;
   const itk::SizeValueType framesToCollect = frames;
+  std::cout << "Version info: " << hwControls.ReadFPGAVersion() << std::endl;
```

Comment:
If this call is necessary, I think we should add it to the "container.SetRFData(bool)" function (calling it when the arg is true).

Documenting it in the app seems insufficient given the importance and perhaps lack of intuitiveness.

Placing this call in the function causes the expected behavior of the SetRFData function.

This is arguable, but I think we will save folks quite some confusion unless we do this.